mangopay/models.py

```diff
@@ -489,6 +489,17 @@ def get_preregistration_data(self):
             "cardRegistrationURL": card_registration.CardRegistrationURL}
         return preregistration_data
 
+    def get_mangopay_card_id(self, registration_data):
+        """
+        function for getting mangopay CardId after having RegistrationData
+        and save the ID to database
+        """
+        client = get_mangopay_api_client()
+        cr = client.cardRegistrations.Get(self.mangopay_id)
```

Comment:
Could you please rename the variables to make the code more readable? `card_registration` is a better name than `cr`